src/app/work-item/work-item-list/work-item-detail/work-item-detail.component.html

```diff
@@ -153,8 +153,29 @@ <h2 *ngIf="loggedIn" id="wi-detail-title-click" class="detail-title truncate">{{
             </span>
           </div>
       </div>
+      <!--Creator-->      
+      <div *ngIf="workItem.relationalData.creator"
+        class="width100 pull-left paddingL20 paddingR20 common-wrap">        
+        <span class="pull-left user-label">Creator</span>
+          <span *ngIf="!workItem.relationalData.creator.attributes.imageURL"
+             class="pull-left fa fa-user-plus user-assign-icon">
+          </span>
+          <img *ngIf="workItem.relationalData.creator.attributes.imageURL"            
+            class="pull-left user-avatar"
+            src="{{workItem.relationalData.creator.attributes.imageURL + '&s=20'}}"
+            id="WI_details_reporter_img" />          
+          <div class="pull-left marginR10">
+            <span *ngIf="!workItem.relationalData.creator.attributes.fullName" id="WI_details_no_reporter_user"
+                class="placeholder"> 
+                Oops we don't know who created this.
+            </span>              
+              <span *ngIf="workItem.relationalData.creator.attributes.fullName" id="WI_details_reporter_user"> 
```

Comment:
Instead putting such check just use ? operator, where you are using it.
{{workItem.relationalData.creator?.attributes.fullName}}